lib/msf/core/opt_raw.rb

```diff
@@ -28,7 +28,7 @@ def normalize(value)
     value
   end
 
-  def valid?(value=self.value)
+  def valid?(value=self.value, check_empty: true)
```

Comment:
Just confirming if this is needed / is there any context on this change? 👀

Reply:
Caught up with Dean, this was just making the API consistent with the other opt implementations of `valid?`